src/main/java/org/folio/innreach/domain/entity/AgencyLocationAcMapping.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.innreach.domain.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.folio.innreach.domain.entity.base.Identifiable;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString(exclude = "localServerMapping")
+@Entity
+@Table(name = "agency_location_ac_mapping")
+public class AgencyLocationAcMapping implements Identifiable<UUID> {
```

Comment:
As far as I know, all entity classes must have audit columns, which means extending Auditable or AbstractEntity classes.

Reply:
The lcs/ac tables are child tables of aggency_location_mapping. The relation is defined with foreign key to parent mapping tables. They all are going to be updated simultaneously using single repository. That's why audit columns are present in parent agency_location_mapping table. Let me know if you think these tables should be auditable anyway

Reply:
Not sure, but I guess all Entity classes should have audit columns. Let's ask @dmtkachenko what he think about this :)

Reply:
I'd say it depends. 
If a child table is very simple entity, like collection of single values, and always included into parent entity, probably doesn't make sense to have separate metadata.
If a child table is more complex and can be concidered on its own then make sense to have a separate metadata. 

Another option is to think about how the entity (parent one and its children) used on UI. If everything always comes together then parent's metadata might be sufficient, and otherwise. If the option is unclear, just add metadata on both levels (parent + child)

Reply:
Ok, thanks for the clarification. Added the audit columns to children tables.